backend/settings/settings.py

```diff
@@ -358,7 +358,7 @@
     },
     "root": {
         "handlers": ["console"],
-        "level": os.environ.get("DJANGO_LOG_LEVEL", "DEBUG"),
+        "level": os.environ.get("DJANGO_LOG_LEVEL", "INFO"),
```

Comment:
I think I found prefer to keep the "DEBUG` level by default here, especially for local environments and CI. It does not seem unreasonable to provide as many details as possible when debugging a failing test case. 

And even in situations where this would be too verbose for developers, `pytest` provides some options to customize this behavior, such as `pytest --show-capture=no` (and variants such as `--show-capture=stdout`) to reduce the amount of visible logs, or `pytest -x` to stop on the first failure.

Reply:
I agree, it's more cautious to keep the DEBUG logs